mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdMapper.java

```diff
@@ -126,7 +126,7 @@ public void save(MailboxMessage mailboxMessage) throws MailboxException {
         mailboxMapper.findMailboxById(mailboxId);
         ComposedMessageIdWithMetaData composedMessageIdWithMetaData = createMetadataFor(mailboxMessage);
         messageDAO.save(mailboxMessage)
-            .thenMany(Flux.merge(
+            .thenMany(Flux.concat(
```

Comment:
I don't really like using `Flux.concat` when we want to express `action1.then(action2)`
What about writing `imapUidDAO.insert(composedMessageIdWithMetaData).then(messageIdDAO.insert(composedMessageIdWithMetaData))` here?